pages/account.tsx

```diff
@@ -44,7 +44,7 @@ const Account: NextPage<Props> = ({ inverters }) => {
           Linking status success: {query.linkSuccess}
         </p>
       )}
-      {inverters.length > 0 ? (
+      {inverters?.length ?? 0 > 0 ? (
```

Comment:
maybe we ditch `??` and just do `inverters?.length` as the logical expression for the ternary here? having `0 > 0` is a bit weird to me. 

```suggestion
      {inverters?.length ?  (
```